Update changelog (#12120)
slint-ui/slint: CHANGELOG.md

 - Fixed per-corner radii for drop shadows
 - Upgraded fontique and parley to 0.10: The `unstable-fontique-09` Cargo feature is renamed to
   `unstable-fontique-010`, and the `slint::fontique_09` module to `slint::fontique_010`.

>  - Get the font size from the desktop settings

Reviewer: ```suggestion
 - The default font size for application is ready from system settings on Windows and Linux.
```

Author: i didn't write it like that originally because it is also on ios and android

 - Added `DragArea` and `DropArea` elements for drag and drop support within a window.
 - Added `data-transfer` type
 - Deprecated calling `init()` explicitly (#11696)
 - Added `Tooltip` element
 - Added `minimized`, `maximized`, `close`, and `hide` on `Window`
 - Added `drop-shadow-spread` and `inner-shadow-{color,blur,offset-x,offset-y,spread}` properties to rectangle. (Only supported in Skia)
 - added `Platform.macos-bring-all-windows-to-front()`

>  - Added `undo()` and `redo()` functions to `TextInput`. 

Reviewer: I think we should follow the style of previous releases and add the element first, so like

```
- `TextInput`: Added `undo()` and `redo()` functions.
```

I think that makes skimming a lot easier, and the more we pack into the release, the more relevant skimming becomes.

 - Fixed percentage size in children impacting parent layout (#3346)
 - Re-evaluate property bindings when a callback handler is changed from native code (#9551)
 - TextInput: allow setting the accessibility value
 - Add landmark accessible roles to `AccessibleRole` (#11831)
 - `animate`: Added `enabled` boolean to toggle animations on/off (defaults to `true`). (#9604)
 - Conversions between `float` and `string` now use the locale's decimal separator,
   which is exposed as `Platform.decimal-separator`. (#10857)

>  - Globals can implement callback aliasing another global callback.

Reviewer: ```suggestion
 - Globals can implement callbacks aliasing another global callback.
```

>  - Added InputType.search

Reviewer: ```suggestion
 - Added `InputType.search` to identify search input fields and inform assistive technologies about their purpose.
```

 - Added `slint::platform::skia_renderer::SkiaWGPURenderer` for rendering into external WGPU textures.
 - Fixed `slint::platform::femtovg_renderer::FemtoVGRenderer` not always being accessible. (#11530)
 - Made `PointerEvent` and `PointerEventKind` types public in `language` module (#11587)

>  - Moved `StandardListViewItem` and `TableColumn` to the `language` module (still re-exported from old location as deprecated)

Reviewer: I'd leave out this entry.

Author: ```suggestion
```

### Tooling

 - SlintPad: Show download progress for the LSP wasm on the splash screen.
 - LSP: `@markdown` completion is now supported.
 - LSP: Fixed selection of imported sub-components in preview.
 - LSP: Update the preview highlight on hover
 - LSP: formatter preserve newlines in expression, as well as enum declarations and export lists.
 - LSP: show runtime warnings in the console (eg: missing image file)
 - Improved file watcher by tracking new file and moved directories.
 - Compiler: Report precise error location within `@markdown` and `@tr` strings. (#11577)
 - Added MCP server feature. (#11542)
 - Viewer can now work on iOS and Android.
 - Added `slint-viewer --remote` to connect to a slint LSP from a mobile device or another device

>  - Added `slint-viewer --screenshot` and `slint-viewer --check`

Author: ```suggestion
 - Added `slint-viewer --remote` to connect to a slint LSP from a mobile device or another device.
 - Added `slint-viewer --screenshot` and `slint-viewer --check`.
```

>  - Added `slint-viewer --screenshot` and `slint-viewer --check`

Reviewer: Trailing period missing here (at least it's inconsistent :)
